test/commands/pull.js

```diff
@@ -1,11 +1,28 @@
 var fs = require('fs');
 var path = require('path');
 
+var remotesToFixtures = [
```

Comment:
nit: When I name variables `xToY`, it's usually a map from things that are `x`s to things that are `y`s